e2e/app/aou-elements/button.ts

```diff
@@ -27,12 +27,13 @@ export default class Button extends BaseElement {
   /**
    * Wait until button is clickable (enabled).
    */
-  async waitUntilEnabled(): Promise<JSHandle> {
-    const handle = this.element.asElement();
-    return await handle.evaluateHandle((e) => {
+  async waitUntilEnabled(): Promise<unknown> {
+    const result = await this.page.waitForFunction((e) => {
```

Comment:
Nit: Could you use a more descriptive variable name than `result`?

Reply:
how about `retrievedValue`?